pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -1129,6 +1129,36 @@ Note ST_Perimeter is only valid for Polygon - use ST_Length for LineString.`,
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+
+	//
+	// Geography modification
+	//
+
+	"st_segmentize": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geography", types.Geography},
+				{"max_segment_length", types.Float},
```

Comment:
late realisation: maybe renaming this to `max_segment_length_meters` helps. now we can omit the "meters" word in the description!

(spelling metres as meters is painful for an aussie but here we are)

Reply:
:laughing: